admin/src/components/UserTable.vue

```diff
@@ -181,6 +233,64 @@ export default {
     }
   },
   methods: {
+    rowDetailsToogle(row, details) {
+      if (
+        this.showConfirmRegisterMailFormular === true ||
+        this.showCreationTransactionListFormular === true
+      ) {
+        this.showCreationFormular = true
+        this.showConfirmRegisterMailFormular = false
+        this.showCreationTransactionListFormular = false
+        return
+      }
+      if (details) {
+        row.toggleDetails()
+      }
+      if (!details) {
+        row.toggleDetails()
+        this.showCreationFormular = true
+        if (this.$refs.showing_detals_true !== undefined) {
+          this.$refs.showing_detals_true.click()
+        }
+      }
```

Comment:
this had issues when switching between two users
1. Open a dialog X for User A
2. Open a dialog Y for User B
-> Problem?